examples/lines_bars_and_markers/bar_label_demo.py

```diff
@@ -18,35 +18,31 @@
 import numpy as np
 
 ###############################################################################
-# Define the data
+# data from https://allisonhorst.github.io/palmerpenguins/
 
-N = 5
-coffee_means = (20, 25, -10, 32, 10)
-tea_means = (30, 13, -14, 21, 17)
-coffee_std = (3, 2, 4, 1, 2)
-tea_std = (4, 3, 2, 3, 5)
-ind = np.arange(N)    # the x locations for the groups
-width = 0.25       # the width of the bars: can also be len(x) sequence
+species = ('Adelie', 'Chinstrap', 'Gentoo')
+sexes = ["Male", "Female"]
+male = (73, 34, 61)
+female = (73, 34, 58)
+sex_counts = [male, female]
+width = 0.6  # the width of the bars: can also be len(x) sequence
 
 ###############################################################################
 # Stacked bar plot with error bars
```

Comment:
```suggestion
```

The text is not correct anymore. Also, it's a bit easier for copy&paste when data and plot commands are in one code block.